Supporting XCTest Framework

csomor opened the pull request:
XCTest Framework is the native testbed for tests on Xcode. This branch will explore whether using the native UI-Testing capabilities could help us run our UI test suite better.

csomor committed:
addding xctest bridge

csomor committed:
attributing source and license

vadz commented:
I'm still not sure what to do about this, but just wanted to ask if the code at https://chromium.googlesource.com/chromium/src/+/master/remoting/host/input_injector_mac.cc (that I've stumbled upon) could help us with fixing the existing `wxUIActionSimulator`?  It looks like everything we want does work for Chrome, somehow.

csomor committed:
fixing include type

csomor committed:
cocoa test project

csomor committed:
renaming dynamic scheme in toplevel project

xcodebuild cannot deal with two identically named schemes in two nested projects

csomor committed:
test xcode builds on travis

csomor committed:
Merge branch 'master' into osx_uitest

csomor committed:
removing DEVELOPMENT_TEAM from project

csomor committed:
fix single test class run

csomor committed:
turning off GUI

csomor committed:
native and console builds, iOS and macOS

csomor committed:
Merge branch 'master' into osx_uitest

csomor committed:
trying to debug script ..

csomor committed:
adding bundle-building for ios

csomor committed:
add check for opened file

csomor committed:
added iOS requirement key

csomor committed:
proper calculation of working directory for resource file access

csomor committed:
failing trad chinese conversions currently hang on iOS

csomor committed:
newly generated makefile with ios bundle support

csomor committed:
another attempt at multi-line

csomor committed:
fixing var

csomor committed:
we must create the bundle folder first

vadz commented:
@csomor Not sure if there is still any chance of something happening with this PR of it should be just closed?

csomor commented:
I'll check whether we could run our "normal" tests under iOS, if not, then this is still relevant ..
